src/tweety/bot.py

```diff
@@ -119,41 +123,43 @@ def iter_tweets(self,  username: Union[str, int, User], pages: int = 1, replies:
 
         :return: (.types.usertweet.UserTweets, list[.types.twDataTypes.Tweet])
         """
-        if wait_time is None:
-            wait_time = 0
+		if wait_time is None:
+			wait_time = 0
 
-        user_id = self._get_user_id(username)
+		user_id = self._get_user_id(username)
 
-        userTweets = UserTweets(user_id, self.request, pages, replies, wait_time, cursor)
+		userTweets = UserTweets(user_id, self.request, pages, replies, wait_time, cursor)
 
-        return userTweets.generator()
+		return userTweets.generator()
 
-    def get_trends(self):
-        """
+	def get_trends(self):
+		"""
         Get the Trends from you locale
 
         :return:list of .types.twDataTypes.Trends
         """
-        trends = []
-        response = self.request.get_trends()
-        for i in response.json()['timeline']['instructions'][1]['addEntries']['entries'][1]['content']['timelineModule']['items']:
-            data = {
-                "name": i['item']['content']['trend']['name'],
-                "url": str(i['item']['content']['trend']['url']['url']).replace("twitter://",
-                                                                                "https://twitter.com/").replace("query",
-                                                                                                                "q"),
-            }
-            try:
-                if i['item']['content']['trend']['trendMetadata']['metaDescription']:
-                    data['tweet_count'] = i['item']['content']['trend']['trendMetadata']['metaDescription']
-            except:
-                pass
-            trends.append(Trends(data))
-        return trends
-
-    @AuthRequired
-    def search(self, keyword: str, pages: int = 1, filter_: str = None, wait_time: int = 2, cursor: str = None):
-        """
+		trends = []
+		response = self.request.get_trends()
+		for i in \
+				response.json()['timeline']['instructions'][1]['addEntries']['entries'][1]['content']['timelineModule'][
+					'items']:
+			data = {
+				"name": i['item']['content']['trend']['name'],
+				"url": str(i['item']['content']['trend']['url']['url']).replace("twitter://",
+																				"https://twitter.com/").replace("query",
+																												"q"),
+			}
+			try:
+				if i['item']['content']['trend']['trendMetadata']['metaDescription']:
+					data['tweet_count'] = i['item']['content']['trend']['trendMetadata']['metaDescription']
+			except:
```

Comment:
If you know which Exception is thrown better to add it here.

Reply:
I am not sure i wrote that code block, you did

Reply:
@philipperemy i think this was written by the maintainer, but its likely a KeyError or IndexError that would occur, i encountered it while adding the features